client/loading/loading-reducer.ts

```diff
@@ -0,0 +1,49 @@
+import { ReadonlyDeep } from 'type-fest'
+import { AUDIO_MANAGER_INITIALIZED, NETWORK_SITE_DISCONNECTED } from '../actions'
+import { immerKeyedReducer } from '../reducers/keyed-reducer'
+
+export interface LoadingState {
+  audio: boolean
+  chat: boolean
+  clientSubscriptions: boolean
+  userSubscriptions: boolean
+  whispers: boolean
+}
+
+const DEFAULT_LOADING_STATE: ReadonlyDeep<LoadingState> = {
+  audio: true,
+  chat: true,
+  clientSubscriptions: true,
+  userSubscriptions: true,
+  whispers: true,
+}
+
+export default immerKeyedReducer(DEFAULT_LOADING_STATE, {
+  [AUDIO_MANAGER_INITIALIZED as any](state: any) {
```

Comment:
why did you make `state` `any` here? The type is already inferred from the default you passed in here, and you're destroying type safety by doing that